### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joshua-deans**

### CODE REVIEW

**Comment by joshua-deans** [comment]
websocket was spelled incorrectly, and Sunday wasn't capitalized.

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Use your key and secret credentials; Find your proper region in aws console (def
 
 
 # Turn on RASA server
-EC2 server can be configured to on and off automatically, Currently turns on every sunday from 1 to 5
+EC2 server can be configured to on and off automatically, Currently turns on every Sunday from 1 to 5
```
Comment (jyleong) [inline README.md]: Good catch, what about "Currently"? Uncapitalize this.

"EC2 server can be configured to on and off automatically"<- I think the grammar here is off

**Review by jyleong — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (joshua-deans) [reply README.md]: I'll make those edits. Thanks.

Comment (joshua-deans) [reply README.md]: Question: is it 1am to 5am or 1pm to 5pm?

Comment (jyleong) [reply README.md]: ah yes, afternoon. 1pm to 5pm

Comment (joshua-deans) [reply README.md]: Perfect. Let me know if you have any more issues with this. Thanks.

Comment (jyleong) [reply README.md]: Great job! Resolving and merging.

**Review by jyleong — APPROVED** [review]

**Pull request merged**